src/async-components/structures/ErrorView.tsx

```diff
@@ -58,7 +58,7 @@ const MobileAppLinks: React.FC<{
     googlePlayUrl?: string;
     fdroidUrl?: string;
 }> = ({ appleAppStoreUrl, googlePlayUrl, fdroidUrl }) => (
-    <Flex gap="var(--cpd-space-6x)">
+    <Flex gap="var(--cpd-space-6x)" className="mx_Flex">
```

Comment:
Override already declared here:
https://github.com/element-hq/element-web/blob/0b6d27053f8c96527fbbecec62bcc31d84ceb071/res/css/structures/ErrorView.pcss#L53-L58

Adding the class manually since the Flex component use a css module.

Reply:
in which case this class should be changed to something scoped to `mx_ErrorView`

Reply:
https://github.com/element-hq/element-web/pull/30357/commits/bc5a78350efdb8997d4a967e105cf77f1f4353e6